### DISCUSSION TITLE
Docker Synology with Grafana

### DISCUSSION THREAD

**Discussion by turkano** [tillsteinbach/VWsFriend#378] [Q&A]
Voy a escribir en español porque creo que la traducción automática será mejor que la mía, pido disculpas.
Estoy intentando que todo funciona en Docker de Synology NAS, para ello instalado 2 imágenes:
![IMG_2684](https://user-images.githubusercontent.com/90693784/208851764-469b483a-6313-4b6e-b760-d95ce4bbf39f.jpeg)
- VWsfriend en localhost:4000 funciona bien, accedo y se conecta al coche sin problemas.
- VWsfriend-grafana localhost:3000 accedo a grafana, pero me muestra todo Sin Datos y la siguiente sugerencia:
![IMG_1223](https://user-images.githubusercontent.com/90693784/208852143-a0050625-4188-4bd3-9bb4-5f5d6d8c183e.jpeg)
Parece que no se conecta a vwfriend en localhost:4000 o que no es capaz de acceder a la base de datos.
He probado a añadir en ADIOTIONAl-PARAMETERS --with-database pero entonces en docker no arranca.
He probado a instalar un docker con postgres, sin exito.
Es verdad que cuando instalo vwsfriend-grafana se me pide rellenar unos parámetros (sino no puedo instalar la imagen), ¿puede que rellene algo mal?
![IMG_2685](https://user-images.githubusercontent.com/90693784/208852585-f9cc7944-c854-487b-88b8-f1da2a445577.jpeg)

![IMG_2686](https://user-images.githubusercontent.com/90693784/208852602-676ed216-b33b-446f-9721-686d98084657.jpeg)

Estoy haciendo algo mal? o simplemente no puede funcionas con grafana en docker de Synology, solo sirve como en el video del WIKI para añadir el coche a HomeKIT?

LOG de Grafana:

![IMG_2687](https://user-images.githubusercontent.com/90693784/208853946-064f440f-0e15-4181-9a19-12390d1f94b6.jpeg)

Muchas gracias.

**Comment by albertusQuelle** [comment]
No one will answer you if you dont post in english
